client/components/layout/internal/Base/__tests__/Base-spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import expectReactShallow from 'expect-react-shallow'
+
+import Base from '..'
+
+describe('Base', () => {
+  it('renders children within a div by default', () => {
+    expectReactShallow(<Base children={<h1>Test</h1>} type='grid-block' />)
+      .to.have.exactly.rendered(<div className='grid-block'><h1>Test</h1></div>)
```

Comment:
This seems to be testing more than what the `it` description says.  For what it says, you could just say `to.have.rendered(<div />)`.

I think it's worth having one test that checks the child rendering, though.  Maybe a separate one?